package.json

```diff
@@ -39,6 +39,8 @@
     "serverless-offline": "^2.2.10",
     "serverless-webpack-plugin": "^0.4.1",
     "sinon": "^1.17.3",
-    "webpack": "^1.13.0"
+    "webpack": "^1.13.0",
+    "crypto": "^0.0.3",
```

Comment:
Do you really need this package? It looks pretty unmaintained and Node has it's own `crypto`